### PR TITLE
fix: Camel case convertor.

### PR DESCRIPTION
**Pull request by prmukherj**

### CODE REVIEW

**Review by hpohekar — APPROVED** [review]

File: tests/test_new_meshing_workflow.py [hunk 1]
```diff
@@ -1220,3 +1221,10 @@ def test_new_meshing_workflow_without_dm_caching(
     with pytest.raises(RuntimeError):
         fault_tolerant.import_cad_and_part_management.arguments()
     assert watertight.import_geometry.arguments()
+
```
Comment (seanpearsonuk) [inline tests/test_new_meshing_workflow.py:1224]: We must add more test cases here.

Comment (prmukherj) [reply tests/test_new_meshing_workflow.py:1224]: Sure @seanpearsonuk. Will do. Could you please suggest some edge cases as well?
Thank you.

Comment (seanpearsonuk) [reply tests/test_new_meshing_workflow.py:1224]: - sequences of upper-case letters at different locations
- sequences of digits at different locations
- underscores at different locations
- starting with lower case
- special characters
- combinations of the above
- single character
- empty strings

Comment (prmukherj) [reply tests/test_new_meshing_workflow.py:1224]: @seanpearsonuk, could you please have a look now?

**Review by seanpearsonuk — APPROVED** [review]
